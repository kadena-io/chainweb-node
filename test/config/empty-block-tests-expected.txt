test-group: new-block
results:
  transactions: []
  minerData: eyJtIjoiTm9NaW5lciIsImtzIjpbXSwia3AiOiI8In0
  transactionsHash: 0SJr05jGkgSIy2cVoCcLDTqGc754cLJ609OAwqnmaXM
  outputsHash: eFkJgs94nLgcU1ic-NJ0pTEjgZvdsN8O66tV-LXb5gQ
  payloadHash: oeRh-MPddEhK69K1QulVmChFwnL49MlXLLKlzLRiSYU
  coinbase: eyJmbENvbW1hbmRSZXN1bHQiOnsicmVzdWx0Ijp7InN0YXR1cyI6InN1Y2Nlc3MiLCJkYXRhIjp7ImkiOm51bGwsImxpdCI6IldyaXRlIHN1Y2NlZWRlZCJ9fSwicGFyZW50SGFzaCI6IktkejA5NXUzNzZaZUZ1c1U1VWpmT2d5YnlaWUZ5ZW0yVklLSXI4QTlZbFkifSwiZmxUeExvZ3MiOlt7InZhbHVlIjp7Imd1YXJkIjp7InByZWQiOiI8Iiwia2V5cyI6W119LCJiYWxhbmNlIjo0Mn0sImtleSI6Ik5vTWluZXIiLCJ0YWJsZSI6IlVTRVJfY29pbl9jb2luLXRhYmxlIn1dfQ
