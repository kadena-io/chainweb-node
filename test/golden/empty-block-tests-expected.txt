test-group: new-block
results:
  transactions: []
  minerData: eyJhY2NvdW50IjoiTm9NaW5lciIsInByZWRpY2F0ZSI6IjwiLCJwdWJsaWMta2V5cyI6W119
  transactionsHash: lL9ztEiU-NwzrlTpBbvhT4M1l5Shsht94OwFyhBaFD0
  outputsHash: XN9f0NgYwskse2b8rQMU3uDo3TJoCFH3RSvo0OnpeQc
  payloadHash: x1rEeYmGXevcitH89DIVAELsVwAibS5F8R_bHgTOPIo
  coinbase: eyJnYXMiOjAsInJlc3VsdCI6eyJzdGF0dXMiOiJzdWNjZXNzIiwiZGF0YSI6IldyaXRlIHN1Y2NlZWRlZCJ9LCJyZXFLZXkiOiJJblZuUVU5bVpqWlphMHR0ZG5CZmNFcEpTSEY0TmtoTVZGSlFkM2xJUlVoRU1tTXlUMnh0Tmt3eVJtY2kiLCJsb2dzIjoibFdaVWZjX0dKdGxUUy1LeDAtYm00aXRfSWE5bnNGZG5jZ2NwSE0td091OCIsIm1ldGFEYXRhIjpudWxsLCJjb250aW51YXRpb24iOm51bGwsInR4SWQiOjd9
